index.html

```diff
@@ -0,0 +1,118 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>WebDev Summit </title>
+    <link rel="stylesheet" href="./assets/css/style.css">
+</head>
+<body>
+        <!-- Navigation Section -->
+        <header>
+            <nav>
+                <div class="mobile-menu">
+                    <button class="mobile-menu-button">&#9776;</button>
+                  </div>
+                  <div class="mobile-menu-options">
+                    <button class="mobile-menu-closing-btn">
+                        &times;
+                    </button>
+                    <ul class="mobile-menu-list-ul">
+                        <li class="mobile-menu-list-li"><a href="./index.html">Home</a></li>
+                        <li class="mobile-menu-list-li"><a href="./about.html">About</a></li>
+                    </ul>
+                  </div>
+                <div class="logo">
+                    <a href="/Capstone-Project/index.html"><img src="./assets/imgs/logo.png" alt="Logo"></a>
+                </div>
+                <div class="menu">
+                    <ul class="menu-list-ul">
+                        <li class="menu-list-li"><a href="./index.html">Home</a></li>
+                        <li class="menu-list-li"><a href="./about.html">About</a></li>
+                    </ul>
+                </div>
+            </nav>
+        </header>
+        <!-- Main Content Section -->
+        <main>
+        <section class="hero-section">
+            <div class="hero-div">
+                <h1 class="thin-text">"Hello! Sharing Knowledge around the Globe"
+                    <br><h1 class="thick-text">Awareness Summit about web development 2023</h1>
+                </h1>
+                <span class="description-about-summit">
+                     A joyful and informative enviroment for sharing creative ideas about the advancements of technology.
+                </span>
+                <span class="summit-date">
+                    2023.10.15(FRI)
+                </span>
+                <span class="summit-location">
+                    @ABC Avenue NY
+                </span>
+            </div>
+        </section>
```

Comment:
- [ ] This section has a background image in the original design. Please add a background to your section too. You are allowed to to have different content but the design should remain the same. 👍🏾